backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/TorClient.kt

```diff
@@ -57,11 +64,31 @@ class TorClient private constructor(
      *
      * Each connection returned by this method is isolated from any other Tor usage.
      */
-    suspend fun connectToLightwalletd(endpoint: String): TorLwdConn =
+    suspend fun createIsolatedWalletClient(endpoint: String): PartialTorWalletClient =
+        accessMutex.withLock {
+            checkNotNull(nativeHandle) { "TorClient is disposed" }
+            IsolatedTorWalletClient.new(
+                isolatedTorClient = isolatedTorClientInternal(),
+                endpoint = endpoint
+            )
+        }
+
+    /**
+     * Connects to the lightwalletd server at the given endpoint.
+     *
+     * Connections used by this client are not isolated from any other Tor usage.
```

Comment:
This is false, and why in a previous review I said that `isolatedTorClient` is confusingly named. The `connectToLightwalletd` JNI method internally uses `isolatedClient()`, so `TorWalletClient`'s connection to lightwalletd is isolated from any other Tor usage:
https://github.com/Electric-Coin-Company/zcash-android-wallet-sdk/blob/6743b74b06a622d33e14e2f6fec7b74a9b9e2a07/backend-lib/src/main/rust/tor.rs#L64-L70

What `TorWalletClient` _doesn't_ do is isolate _individual queries to the lightwalletd server over this connection_ from each other.
```suggestion
     * This client is isolated from any other Tor usage. Queries made with this client are
     * not isolated from each other; use `createIsolatedWalletClient()` if you need this.
```